src/main.cpp

```diff
@@ -723,13 +723,14 @@ static int parse_confs(ConfFile &conf)
         char *addr;
         bool server;
         unsigned long port;
-        char *filter;
+        char *msgIdFilter;
     };
     static const ConfFile::OptionsTable option_table_udp[] = {
         {"address", true, ConfFile::parse_str_dup, OPTIONS_TABLE_STRUCT_FIELD(option_udp, addr)},
         {"mode", true, parse_mode, OPTIONS_TABLE_STRUCT_FIELD(option_udp, server)},
         {"port", false, ConfFile::parse_ul, OPTIONS_TABLE_STRUCT_FIELD(option_udp, port)},
-        {"filter", false, ConfFile::parse_str_dup, OPTIONS_TABLE_STRUCT_FIELD(option_udp, filter)},
+        {"AllowMsgIdOut", false, ConfFile::parse_str_dup,
+         OPTIONS_TABLE_STRUCT_FIELD(option_udp, msgIdFilter)},
```

Comment:
Having both lines write to the msgIdFilter, the newer option will overwrite the old one, if both are present in the config file. Seems to be a good solution since unknown config keys are ignored. So a config with both `Filter` and `AllowMsgIdOut` will work in any version.